inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/iceberg/IcebergSinkDTO.java

```diff
@@ -58,7 +59,7 @@ public class IcebergSinkDTO {
     private String dataPath;
 
     @ApiModelProperty("File format, support: Parquet, Orc, Avro")
-    private String fileFormat;
+    private FileFormat fileFormat;
```

Comment:
If the type of `fileFormat` changed to `Enum`, the `getFromJson()` method may be lost `fileFormat` field.

Reply:
Reverted. No change in this pr.